CHANGELOG.md

```diff
@@ -10,46 +10,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Added Cmd/Ctrl + Enter keyboard shortcut that allows to fill the selected range of cells with the value of the active cell. [#9366](https://github.com/handsontable/handsontable/issues/9366)
-- Adds support for Home/End and fix buggy behavior of the Cmd/Ctrl+Z/Shift+Z keyboard shortcuts. [#9367](https://github.com/handsontable/handsontable/issues/9367)
-- Add Arabic Language [#9208](https://github.com/handsontable/handsontable/issues/9208)
+### Fixed
+- Fixed an issue where cell filtering used formula results instead of source values. [#5455](https://github.com/handsontable/handsontable/issues/5455)
+- Fixed an issue where the Handsontable documentation didn't build locally on Windows. [#8677](https://github.com/handsontable/handsontable/issues/8677)
+- Fixed a wrong TypeScript definition in the `BasePlugin` class. [#9175](https://github.com/handsontable/handsontable/issues/9175)
+- Fixed an issue where using `updateSettings()` caused configuration options to disappear. [#6552](https://github.com/handsontable/handsontable/issues/6552)
+- Fixed an issue where using `updateSettings()` caused hidden columns to reappear. [#7165](https://github.com/handsontable/handsontable/issues/7165)
+- Fixed an issue where using `updateSettings()` caused merged cells to unmerge. [#3315](https://github.com/handsontable/handsontable/issues/3315)
+- Fixed an issue where using `updateSettings()` caused the state of nested rows to reset. [#8838](https://github.com/handsontable/handsontable/issues/8838)
+- Fixed an issue where using `updateSettings()` caused problems with column sorting. [#7688](https://github.com/handsontable/handsontable/issues/7688)
+- Fixed an issue where the <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>Z</kbd> and <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Z</kbd> keyboard shortcuts didn't behave properly inside cell editors. [#9367](https://github.com/handsontable/handsontable/issues/9367)
+- Fixed an issue where the <kbd>Page Up</kbd> and <kbd>Page Down</kbd> keys didn't behave properly in context menus. [#9394](https://github.com/handsontable/handsontable/issues/9394)
+- Fixed an issue where the dropdown menu contained unwanted rectangle elements. [#9240](https://github.com/handsontable/handsontable/issues/9240)
+- *Vue 2:* Fixed an issue where repeatedly changing Handsontable's height through `updateSettings()` caused Handsontable to crash. [#7546](https://github.com/handsontable/handsontable/issues/7546)
+- *Vue 2:* Fixed an issue where the `failed` validation status got erased when editing a new row. [#7541](https://github.com/handsontable/handsontable/issues/7541)
+- *React:* Fixed an issue where using `setState()` within the `afterFilter` hook broke filtering. [#7567](https://github.com/handsontable/handsontable/issues/7567)
+- *React, Angular, Vue 2, Vue 3:* Fixed an issue where using `updateSettings()` caused problems for state managers. [#8372](https://github.com/handsontable/handsontable/issues/8372)
```

Comment:
This is very important and is mentioned in the blog post. It should be moved up in the changelog.

Reply:
Ok. Since we don't list the framework issues at the bottom anymore, I'm moving up all issues related to the `updateSettings()` changes: [87456d9](https://github.com/handsontable/handsontable/pull/9271/commits/87456d93f74abc791720357ea8f7861581b74837) ([output](https://github.com/handsontable/handsontable/blob/release/12.0.0-docs/CHANGELOG.md#fixed))